programs/nft-voter/src/tools/token_metadata.rs

```diff
@@ -1,19 +1,18 @@
 use anchor_lang::prelude::*;
-
-use mpl_token_metadata::state::{Metadata, TokenMetadataAccount};
+use mpl_token_metadata::accounts::Metadata;
 
 use crate::error::NftVoterError;
 
 pub fn get_token_metadata(account_info: &AccountInfo) -> Result<Metadata> {
     if *account_info.owner != mpl_token_metadata::ID {
         return Err(NftVoterError::InvalidAccountOwner.into());
     }
-
-    let metadata = Metadata::from_account_info(account_info)?;
+    let metadata_data_bytes = &account_info.data.borrow() as &[u8];
+    let metadata = Metadata::from_bytes(metadata_data_bytes)?;
```

Comment:
nit: this can be replaced with `try_from` 

```
use std::convert::TryFrom;

let metadata = Metadata::try_from(account_info)?;
```

Reply:
TIL. updated the code sir.